packages/@textlint/fixer-formatter/src/formatters/raw-output.ts

```diff
@@ -0,0 +1,10 @@
+// LICENSE : MIT
+"use strict";
+import type { TextlintFixResult } from "@textlint/types";
+
+export default function (results: TextlintFixResult[]) {
+    let output = "";
+    results.forEach((result) => (output += `\n${result.output}`));
```

Comment:
```suggestion
    results.forEach((result) => (output += result.output));
```

I think that We can not add new line.
It will change file even though there are not changes.
If textlint lint a file and does not found any error, the output should be same with input file.



Similar example: `cat` just combine a and b.

```
cat a.txt b.txt
```

Reply:
Nice catch. Thanks. https://github.com/textlint/textlint/pull/978/commits/0776215a49d47fbe050a11dea80d5b2780381a9a


```sh
// After the change
>>> diff old.md new.md
```